Resources/config/routing.yml

```diff
@@ -122,6 +122,10 @@ canal_tp_mtt_block_delete:
         externalNetworkId: -?[a-zA-Z0-9_\-:]*
 
 #Calendar
+canal_tp_mtt_calendar_create:
+    pattern: /networks/{externalNetworkId}/calendars/create
```

Comment:
you can remove externalNetworkId as a calendar is available for all the networks